### PR TITLE
[VIT-6388] Docs + Changelog: Team scope requirements

### PR DESCRIPTION
**Pull request by ItachiEU**

### CODE REVIEW

File: docs/api-reference/org-management/team-scope-requirements/upsert-team-scope-requirements.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+openapi: post /v1/org/{org_id}/team/{team_id}/scope_requirements
+title: Set / Reset Team ETL Pipelines
```
Comment (andersio) [inline docs/api-reference/org-management/team-scope-requirements/upsert-team-scope-requirements.mdx:3]: ditto

File: docs/changelog/wearables/api.mdx [hunk 2]
```diff
@@ -2,6 +2,24 @@
 title: "API"
 ---
 
+### Team Scope Requirements (Apr 24)
+You can now tailor your Vital Team's OAuth scopes using the [Org Management API](/api-reference/org-management/team-scope-requirements/upsert-team-scope-requirements)
+to define the necessary permissions for a user's connection to be successful.
```
Comment (andersio) [inline docs/changelog/wearables/api.mdx:7]:
```suggestion
You can now tailor what scopes Vital would request from your Users using the [Org Management API](/api-reference/org-management/team-scope-requirements/upsert-team-scope-requirements). This applies when your user connects to an [OAuth provider](/wearables/connecting-providers/auth_types) that support scopes.
```

File: docs/changelog/wearables/api.mdx [hunk 3]
```diff
@@ -2,6 +2,24 @@
...
+
+<Accordion icon="lightbulb" iconType="duotone" title="Details" defaultOpen>
+When the `user_must_grant` list under Team Scope Requirements setting is not empty, Vital ensures that a user's connection succeeds only if all the required scopes are granted.
```
Comment (andersio) [inline docs/changelog/wearables/api.mdx:10]:
```suggestion
Vital ensures that any new provider connection can establish only when the user has granted all the scopes you specified in the `user_must_grant` scope list.
```

File: docs/changelog/wearables/api.mdx [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
+
+The scopes specified under `user_may_grant`, will be visible for the user to opt in to, but without affecting the status of their connection.
```
Comment (andersio) [inline docs/changelog/wearables/api.mdx:12]:
```suggestion
The scopes specified as `user_may_grant` would be requested alongside the `user_must_grant` scopes. However, unlike `user_must_grant`, they do not prevent the connection from being established.
```

File: docs/api-reference/org-management/team-scope-requirements/upsert-team-scope-requirements.mdx (hunk 1 above)
Comment (andersio) [reply docs/api-reference/org-management/team-scope-requirements/upsert-team-scope-requirements.mdx:3]: just "Set" is fine imo

**Review by piresrui — APPROVED** [review]

**Pull request merged**